misc_docs/syntax/decorator_uncurry.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: "uncurry-decorator"
+keywords: ["uncurry", "decorator"]
+name: "@uncurry"
+summary: "This is the `@uncurry` decorator."
+category: "decorators"
+---
+
+The `@uncurry` decorator allows you to annotate **external** function callback argument types
```

Comment:
The `@uncurry` decorator can be used to mark any callback argument within an **external** function as an uncurried function without the need for any [explicit uncurried function syntax](/docs/manual/latest/function#uncurried-function) (`(.) => { ... }`).

Reply:
Thanks, updated.